src/tasks/task-se3-equality.cpp

```diff
@@ -199,8 +206,18 @@ namespace tsid
       m_v_ref_vec = m_v_ref.toVector();
       m_v = v_frame.toVector();
 
-      m_constraint.setMatrix(m_mask.asDiagonal() * m_J);
-      m_constraint.setVector(m_mask.asDiagonal() * (m_a_des - m_drift.toVector()));
+
+      m_a_des -= m_drift.toVector();
```

Comment:
This is a bad idea because now the vector `m_a_des` does not contain the desired acceleration, which the user can read through the method `getDesiredAcceleration`. It'd be better to compute the difference `m_a_des - m_drift` directly in line 217.